code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1903,9 +1904,13 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		detach()
 		return
 
+/obj/structure/centrifuge/AltClick()

```

Comment:
Maybe a checkhotkeys proc, but then again it'd be completely redundant since there's only ctrlclick and altclick, both of which you can immediately try